### ISSUE TITLE
Doesn't work with XCode 6.1.1

### ISSUE THREAD

**Issue by Quapper** [Galeas/CocoaPodUI#27]
When I create a new project and try to set this up it just closes my project and nothing else happens!

**Comment by huntedman** [comment]
Yep. I can confirm similar problem. Freezes at integrating project.

**Comment by Quapper** [comment]
Yes, mine also freezes at this stage now!

**Comment by chintan1891** [comment]
@DenninDalke @huntedman @Quapper Please checkout this fork "https://github.com/chintan1891/CocoaPodUI" and let me know if it resolves your issue or not.
